src/api/constants.ts

```diff
@@ -322,4 +320,5 @@ export const VC_REMOVE_ORIGINAL_FIELDS = false
 // Auth
 export const PROPOSAL_MESSAGE_TITLE = 'AuthRequest'
 
-export const REPLY_PROTECTION_INTERVAL = 30
+// since date in javascript is always millisecond
+export const REPLY_PROTECTION_INTERVAL = 30 * 1000
```

Comment:
@jay-dee7 Let's consider halving this to 15''.

Reply:
done